inference-engine/src/mkldnn_plugin/nodes/mkldnn_roi_pooling_node.cpp

```diff
@@ -677,13 +659,43 @@ void MKLDNNROIPoolingNode::execute() {
                 }
             }
 
-            if (roi_pooling_kernel) {
-                (*roi_pooling_kernel)(&arg);
+            if (execPtr) {
+                execPtr->exec(arg);
             }
         }
     });
 }
 
+void MKLDNNROIPoolingNode::executeDynamicImpl(mkldnn::stream strm) {
+    return execute(strm);
+}
+
+void MKLDNNROIPoolingNode::prepareParams() {
+    execPtr = std::make_shared<ROIPoolingExecutor>(jpp);
```

Comment:
also on `prepareParams` we need versify input/output memory objects
https://github.com/openvinotoolkit/openvino/blob/2d27b73de96a9ff4f73145ca67c1680e1977930a/inference-engine/src/mkldnn_plugin/nodes/mkldnn_reorder_node.cpp#L105-L112

Reply:
Verifying is added, thanks!

Reply:
looks like we still need to add reference executor where we will be store `jpp`
otherwise here you got exception `IE_THROW() << "Can't create jit RoiPooling kernel";`

Reply:
Discussed offline: reference executor will be created as part of the 70671 issue. At the moment, reference execution used the class field `jit_roi_pooling_params refParams`.
Code changed to force reference execution in case when it is impossible to use JIT executor.
Thank you!